### PR TITLE
Update changelog for v47.5.0

### PR DESCRIPTION
**Pull request by brpowell**
### What does this PR do?

Update changelog for v47.5.0

### CODE REVIEW

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,17 @@
+# 47.5.0 - November 7, 2019
+
+## Fixed
+
+#### salesforcedx-vscode-core
+
+- Update Org Browser to use default org when the view is opened ([PR #1763](https://github.com/forcedotcom/salesforcedx-vscode/pull/1763))
```
Comment (lcampos) [inline packages/salesforcedx-vscode/CHANGELOG.md]:
```suggestion
- Update Org Browser to use default org when the view is opened ([PR #1763](https://github.com/forcedotcom/salesforcedx-vscode/pull/1763), [Issue #1761](https://github.com/forcedotcom/salesforcedx-vscode/issues/1761))
```

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,17 @@
...
+
```
Comment (sfsholden) [inline packages/salesforcedx-vscode/CHANGELOG.md]: Just an fyi - I am just about done with a QA for the apex LSP changes that Roy demo'd last week. We are trying to get them in for tomorrow too.

Comment (brpowell) [reply packages/salesforcedx-vscode/CHANGELOG.md]: Cool, lmk when the jar is merged and I'll update the log

Comment (brpowell) [reply packages/salesforcedx-vscode/CHANGELOG.md]: Cool, lmk when the jar is merged and I'll add it to the log

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,17 @@
+# 47.5.0 - November 7, 2019
+
+## Fixed
+
+#### salesforcedx-vscode-core
+
+- Update Org Browser to use default org when the view is opened ([PR #1763](https://github.com/forcedotcom/salesforcedx-vscode/pull/1763), [Issue #1761](https://github.com/forcedotcom/salesforcedx-vscode/issues/1761))
```
Comment (hbc-bindu) [inline packages/salesforcedx-vscode/CHANGELOG.md]: Rephrase as:
Update Org Browser to use the current default org and display the correct alias in the org picker

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,17 @@
...
+
+## Added
+
+#### docs
+
+- Add code formatting to command help for LWC [local development](https://developer.salesforce.com/tools/vscode/en/lwc/localdev) ([PR #1722](https://github.com/forcedotcom/salesforcedx-vscode/pull/1722))
```
Comment (hbc-bindu) [inline packages/salesforcedx-vscode/CHANGELOG.md]: Rephrase as:
Add code formatting to the command to view LWC local development help.

**Pull request merged**